server/server.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const db = require('./db.js');
+const port = 1337;
+
+app.use(express.static('./public/dist'));
+
+app.get('/api/users', (req, res) => {
+  db.User.findAll()
```

Comment:
No * !!!! Set a limit for items returned.

Reply:
I thought it would be ok here just because I'm not dealing with a huge amount of data, but it's generally not good practice, huh